nucliadb/nucliadb/vectorsets/datamanager.py

```diff
@@ -0,0 +1,104 @@
+# Copyright (C) 2021 Bosutech XXI S.L.
```

Comment:
Creating a module vectorset with a datamanager feels weird as we already have a datamanagers module

Reply:
👍🏼 will put it in common/datamanagers.
I did it this way to start to avoid banch conflicts with the matryoshka PR